yaooqinn/kyuubi
# [KYUUBI #291] Add a quick start guide for DBeaver

![yaooqinn](https://badgen.net/badge/Hello/yaooqinn/green) [![PR 291](https://badgen.net/badge/Preview/PR%20291/blue)](https://github.com/yaooqinn/kyuubi/pull/291) [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)



### Please add issue ID here?

fixes #291

### Why are the changes needed?


add documentation for dbeaver database tool

### Test Plan:
- Add some test cases that check the changes thoroughly including negative and positive cases if possible
- Add screenshots for manual tests if appropriate
![image](https://user-images.githubusercontent.com/8326978/104806362-d9669880-5811-11eb-9e9f-97d630c345a7.png)

- [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request
